### DISCUSSION TITLE
BT-13713 forbidden in notice type 28 but gives validation error

### DISCUSSION THREAD

**Discussion by EEanalyst** [OP-TED/eForms-SDK#316] [Q&A] [answered]
Validating notice type 28 we get an error:
"For a DAP, Result, Contract Modification or Completion notice, every existing Lot (Lot (BT-137-Lot)) must be associated with a LotResult (Result Lot Identifier (BT-13713-LotResult))"

If we check Fields.json, it says BT-13713 is forbidden for notice type 28:
```
}, {
    "id" : "BT-13713-LotResult",
    "parentNodeId" : "ND-LotResult",
    "name" : "Result Lot Identifier",
    "btId" : "BT-13713",
    "xpathAbsolute" : "/*/ext:UBLExtensions/ext:UBLExtension/ext:ExtensionContent/efext:EformsExtension/efac:NoticeResult/efac:LotResult/efac:TenderLot/cbc:ID",
    "xpathRelative" : "efac:TenderLot/cbc:ID",
    "type" : "id-ref",
    "idSchemes" : [ "LOT" ],
    "legalType" : "IDENTIFIER",
    "repeatable" : {
      "value" : false,
      "severity" : "ERROR"
    },
    "forbidden" : {
      "value" : false,
      "severity" : "ERROR",
      "constraints" : [ {
        "noticeTypes" : [ "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "20", "21", "22", "23", "24", "28", "CEI", "T01", "X01", "X02" ],
        "value" : true,
        "severity" : "ERROR"
      } ]
```
How can we get past this error?

**Comment by bertrand-lorentz** [comment, marked as answer]
Which version of the SDK are you using for your notice?
How are you doing the validation (using CVS or running the Schematron files) ?
It seems the contradiction you describe is not present in SDK 1.5.

**Reply by EEanalyst** [reply]
We are currently on version 1.3.2 and use CVS to validate.
According to fields.json I see that for BT-13713 notice type 28 is forbidden in both 1.3.2 and 1.5.1 versions

**Reply by bertrand-lorentz** [reply]
The inconsistency was corrected in SDK 1.4 : the rule BR-BT-00137-0209 that corresponds to the error you get ("...every existing Lot (Lot (BT-137-Lot)) must be associated with a LotResult (Result Lot Identifier (BT-13713-LotResult))" has been modified to not cause an error if the notice does not have any LotResult.

We've also just added a new example for notice subtype 28 : https://github.com/OP-TED/eForms-SDK/blob/develop/examples/notices/veat_23.xml
